src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,88 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type PropTypes = {
+  totalItems: number;
+  itemsPerPage: number;
+  currentPage: number;
+  paginate: (pageNumber: number) => void;
+  setCurrentPage: (newPage: number) => void;
+};
+
+export const Pagination: React.FC<PropTypes> = ({
+  itemsPerPage,
+  totalItems,
+  paginate,
+  currentPage,
+  setCurrentPage,
+}) => {
+  const pageNumbers = [];
+  const pages = Math.ceil(totalItems / itemsPerPage);
+
+  // eslint-disable-next-line no-plusplus
+  for (let i = 1; i <= pages; i++) {
+    pageNumbers.push(i);
+  }
```

Comment:
you can calculate it in `pageNumbers` declaration using `useMemo` and `pages.map`